src/dataflow/internal/process/functions/function-call.ts

```diff
@@ -107,6 +110,32 @@ export function processFunctionCall<OtherInfo>(functionCall: RFunctionCall<Other
 		inIds.push(...functionName.in, ...functionName.unknownReferences)
 	}
 
+	// parse a source call and analyze the referenced code
+	if(named && functionCallName == 'source') {
```

Comment:
this fails, whenever source is overwritten, you have to check if the function name really resolves to the builtin `source` for the current environment.

Reply:
oh, I think I thought that's what `named` did

Reply:
named indicates that the function is not anonymous (e.g. the second call in ` produceCounter(0)()`).

Reply:
this should probably be part of a comment